lib/Evaluate/formatting.cpp

```diff
@@ -14,13 +14,13 @@
 #include "flang/Evaluate/tools.h"
 #include "flang/Parser/characters.h"
 #include "flang/Semantics/symbol.h"
-#include <sstream>
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Interesting that it included string stream.

Reply:
I believe that the reason was because of this function:
std::string ExpressionBase<RESULT>::AsFortran() const {
  std::ostringstream ss;
I've removed with the new commit.